src/components/Directory/Filter/index.tsx

```diff
@@ -60,29 +60,30 @@ export default function Filter({
       <button
         ref={industryDropdownButtonRef}
         onClick={() => setIsIndustryDropdownOpen(!isIndustryDropdownOpen)}
-        className={`flex w-full items-center justify-between bg-brandGold px-4 py-2 transition-all duration-300 ease-out hover:shadow-lg md:h-12 md:w-1/2 dark:text-black ${
+        className={`relative flex w-full items-center justify-between bg-brandGold px-4 py-2 transition-all duration-300 ease-out hover:shadow-lg md:h-12 md:w-1/2 dark:text-black ${
           isIndustryDropdownOpen
             ? "rounded-t-lg font-semibold"
             : "rounded-lg font-normal"
-        } `}
+        }`}
       >
         <div className="flex items-center">
-          <FilterIcon></FilterIcon>
+          <FilterIcon />
           <span className="ml-2">Filter by Industry</span>
         </div>
         <div
           className={`ml-2 transition-opacity duration-300 ease-out ${
             isIndustryDropdownOpen ? "opacity-100" : "opacity-0"
           }`}
         >
-          <XIcon></XIcon>
+          <XIcon />
         </div>
       </button>
 
+      {/* Fixed Height Chip Container */}
       <div
-        className={`flex w-full flex-wrap gap-2 ${
-          selectedIndustries.length === 0 ? "mt-0" : "mt-4"
-        } ${isIndustryDropdownOpen ? "hidden" : "block"}`}
+        className={`flex h-16 w-full min-w-[300px] flex-wrap gap-2 transition-opacity duration-300 ease-out ${
+          isIndustryDropdownOpen ? "invisible" : "visible"
+        }`}
```

Comment:
Interesting approach. Did we want to make the items in here invisible? I thought we'd just leave them up since they're visibly blocked from view by the dropdown. What do you think @countaight, @geraxrodriguez, @jvmontes? If we decide to not hide/show, we could use the suggested code changes below to just have it appear permanently whether it's open or not.

```suggestion
        className={`flex h-16 w-full min-w-[300px] flex-wrap gap-2`}
```

Reply:
Actually. I realize now that this is for the actual dropdown and not hiding and showing the tags. You can disregard that part. But I'm curious if we want animation here or not. I won't block this PR on whether or not it has animations though.

Reply:
> When I remove the line of code making the chips container invisible, it looks fine, except that the filter dropdown animates from the bottom left, and just looks a little off. I wish it would actually _drop down_ from the filter button in terms of the animation. @countaight you said you were good at animations, any insights around this would be much appreciated!

@countaight nevermind, I got it to work nicely!